tmt/steps/prepare/install.py

```diff
@@ -238,3 +246,21 @@ def go(self, guest):
             guest.execute(
                 f"{check} || {command} install {options} "
                 f"{packages}{yum_check}")
+
+        # Install debug{info,source} from repos
+        if repo_debuginfos:
+            # Show a brief summary by default
+            if not self.opt('verbose'):
+                summary = fmf.utils.listed(repo_debuginfos, max=3)
+                self.info('package', summary, 'green')
+            # Provide a full list of packages in verbose mode
+            else:
+                summary = fmf.utils.listed(repo_debuginfos, 'package')
+                self.info('package', summary + ' requested', 'green')
+                for package in sorted(repo_debuginfos):
+                    self.verbose(package, shift=1)
+            # Quote package names and prepare the rpm check
+            packages = ' '.join(
+                [tmt.utils.quote(package) for package in repo_debuginfos])
+            check = f'rpm -q --whatprovides {packages}'
+            guest.execute(f"{check} || debuginfo-install -y {repo_debuginfos}")
```

Comment:
I think the `debuginfo-install` command does not always have to be installed on the box. We should probably include `{command} install -y /usr/bin/debuginfo-install` to ensure the command is available.

Reply:
Done, thanks.